src/common/constants/instruction-step.js

```diff
@@ -1,6 +1,26 @@
-export const COVID_19 = "https://gateway.pinata.cloud/ipfs/QmcEASf8BoztaFUKA68dmRQrECUakadmjEBwZXKQGroMaS"
-export const DRIED_BLOOD = "https://gateway.pinata.cloud/ipfs/QmPZ5Ew7dZbLpjfJxLTgmYWEUycHVsLLDxaRD5RwPn2oQs"
-export const URINE_COLLECTION = "https://gateway.pinata.cloud/ipfs/Qmcei89XiFjZuU5dEnJ9XkDB9JHhisaY5cuz2Ym66mMRhc"
-export const FECAL_COLLECTION = "https://gateway.pinata.cloud/ipfs/QmaHSSVsLZarX4EJbhWdNHjTnehpFMjh68X4mv1EiFSEaB"
-export const SALIVA_COLLECTION = "https://gateway.pinata.cloud/ipfs/QmPZ5Ew7dZbLpjfJxLTgmYWEUycHVsLLDxaRD5RwPn2oQs"
-export const BUCCAL_COLLECTION = "https://gateway.pinata.cloud/ipfs/QmcSZrMdraU2RKXhXbGBTZ8p1CnQibixeqDtc6Kwun7Qd2"
+export default [
+  {
+    name: "Covid 19 Saliva Test",
+    link: "https://gateway.pinata.cloud/ipfs/QmcEASf8BoztaFUKA68dmRQrECUakadmjEBwZXKQGroMaS"
+  },
+  {
+    name: "Blood Cells - Dried Blood Spot Collection Process",
+    link: "https://gateway.pinata.cloud/ipfs/QmPZ5Ew7dZbLpjfJxLTgmYWEUycHVsLLDxaRD5RwPn2oQs"
+  },
+  {
+    name: "Epithelial Cells - Buccal Swab Collection Process",
+    link: "https://gateway.pinata.cloud/ipfs/QmcSZrMdraU2RKXhXbGBTZ8p1CnQibixeqDtc6Kwun7Qd2"
+  },
+  {
+    name: "Fecal Matters - Stool Collection Process",
+    link: "https://gateway.pinata.cloud/ipfs/QmaHSSVsLZarX4EJbhWdNHjTnehpFMjh68X4mv1EiFSEaB"
+  },
+  {
+    name: "Saliva - Saliva Collection Process",
+    link: "https://gateway.pinata.cloud/ipfs/QmPZ5Ew7dZbLpjfJxLTgmYWEUycHVsLLDxaRD5RwPn2oQs"
+  },
+  {
+    name: "Urine - Clean Catch Urine Collection Process",
+    link: "https://gateway.pinata.cloud/ipfs/Qmcei89XiFjZuU5dEnJ9XkDB9JHhisaY5cuz2Ym66mMRhc"
+  }
+]
```

Comment:
just curious, why did you decide to use an array instead of an object? because I think if you decide to use an array instead of an object, it will take a lot of time to get 1 specific object by filtering using `.filter()`

Reply:
thanks for point it out mas, I have changed this to object.